packages/story-editor/src/components/library/panes/pageTemplates/templateSave.js

```diff
@@ -178,18 +181,33 @@ function TemplateSave({ setShowDefaultTemplates, updateList }) {
   );
 
   return (
-    <SaveButton
-      aria-disabled={isDisabled}
-      onClick={handleSaveTemplate}
-      $isDisabled={isDisabled}
-    >
-      <IconWrapper>
-        <Icon aria-hidden />
-      </IconWrapper>
-      <StyledText size={TextSize.Small}>
-        {__('Save current page as template', 'web-stories')}
-      </StyledText>
-    </SaveButton>
+    <>
+      <SaveButton
+        aria-disabled={isDisabled}
+        onClick={() => {
+          !isDisabled && setIsDialogOpen(true);
```

Comment:
We don't typically use this code style elsewhere.

```suggestion
          if (!isDisabled) {
            setIsDialogOpen(true);
          }
```